operations/policies/policies.rego

```diff
@@ -8,7 +8,7 @@ should_be_namespaced(contents) {
 }
 
 should_be_namespaced(contents) {
-	not contents.kind in ["PodSecurityPolicy", "Namespace"]
+	not contents.kind in ["PodSecurityPolicy", "Namespace", "CustomResourceDefinition"]
```

Comment:
Note to reviewers: `CustomResourceDefinition` doesn't have `namespace` because it's a global resource.